### PR TITLE
Chore/update filter sort table editor to new popover

### PR DESCRIPTION
**Pull request by joshenlim**

### CODE REVIEW

File: apps/studio/components/grid/components/header/filter/FilterPopover.tsx [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
 import { isEqual } from 'lodash'
+import { FilterIcon, Plus } from 'lucide-react'
 import { KeyboardEvent, useCallback, useMemo, useState } from 'react'
-import { Button, IconFilter, IconPlus, Popover } from 'ui'
 
 import { formatFilterURLParams } from 'components/grid/SupabaseGrid.utils'
 import type { Filter, SupaTable } from 'components/grid/types'
+import {
+  Button,
+  Popover,
```
Comment (MildTomato) [inline apps/studio/components/grid/components/header/filter/FilterPopover.tsx:11]: Popover_shadcn

File: apps/studio/components/grid/components/header/filter/FilterPopover.tsx [hunk 2]
```diff
@@ -16,38 +23,55 @@ export interface FilterPopoverProps {
...
-          </div>
-        }
-      >
-        <span>{btnText}</span>
-      </Button>
-    </Popover>
+    <Popover_Shadcn_ open={open} onOpenChange={setOpen} modal={false}>
+      <PopoverTrigger_Shadcn_ asChild>
+        <Button
+          asChild
```
Comment (MildTomato) [inline apps/studio/components/grid/components/header/filter/FilterPopover.tsx:52]: `<span>` ?

File: apps/studio/components/grid/components/header/sort/SortPopover.tsx [hunk 3]
```diff
@@ -16,38 +23,49 @@ export interface SortPopoverProps {
...
-          </div>
-        }
-      >
-        <span>{btnText}</span>
-      </Button>
-    </Popover>
+    <Popover_Shadcn_ modal={false} open={open} onOpenChange={setOpen}>
+      <PopoverTrigger_Shadcn_ asChild>
+        <Button
+          asChild
```
Comment (MildTomato) [inline apps/studio/components/grid/components/header/sort/SortPopover.tsx:46]: ??

**Review by MildTomato — CHANGES_REQUESTED** [review]

File: apps/studio/components/grid/components/header/filter/FilterPopover.tsx [hunk 4]
```diff
@@ -131,10 +138,14 @@ const FilterOverlay = ({ table, filters: filtersFromUrl, setParams }: FilterOver
       </div>
       <Popover.Separator />
```
Comment (MildTomato) [inline apps/studio/components/grid/components/header/filter/FilterPopover.tsx:139]: Can you add a separator into the shadcn/popover?, can copy what is used in shadcn/Dropdown

<PopoverSeperator/>

**Review by MildTomato — CHANGES_REQUESTED** [review]

File: apps/studio/components/grid/components/header/filter/FilterPopover.tsx (hunk 4 above)
Comment (MildTomato) [reply apps/studio/components/grid/components/header/filter/FilterPopover.tsx:139]: https://github.com/supabase/supabase/blob/02c0d75dfbe5af5260376a009bbf14e675d39b2d/packages/ui/src/components/shadcn/ui/dropdown-menu.tsx#L152-L162

Comment (MildTomato) [reply apps/studio/components/grid/components/header/filter/FilterPopover.tsx:139]: oh nice, the links show previews now ❤️

Comment (MildTomato) [reply apps/studio/components/grid/components/header/filter/FilterPopover.tsx:139]: actually your probably better copying [this one](https://github.com/supabase/supabase/blob/master/packages/ui/src/components/shadcn/ui/dialog.tsx/#L200-L206)

Comment (MildTomato) [reply apps/studio/components/grid/components/header/filter/FilterPopover.tsx:139]: https://github.com/supabase/supabase/blob/master/packages/ui/src/components/shadcn/ui/dialog.tsx/#L200-L206

Comment (joshenlim) [reply apps/studio/components/grid/components/header/filter/FilterPopover.tsx]: gotcha!

File: apps/studio/components/grid/components/header/sort/SortPopover.tsx (hunk 3 above)
Comment (joshenlim) [reply apps/studio/components/grid/components/header/sort/SortPopover.tsx]: removed!

File: apps/studio/components/grid/components/header/filter/FilterPopover.tsx (hunk 2 above)
Comment (joshenlim) [reply apps/studio/components/grid/components/header/filter/FilterPopover.tsx]: removed!

**Review by alaister — APPROVED** [review]
LGTM!

**Review by MildTomato — APPROVED** [review]

**Pull request merged**